roles/node/vars/ufw_rules.yml

```diff
@@ -0,0 +1,6 @@
+node_ufw_rules:
+  - port: "{{ avail[type].ports.p2p_port }}"
```

Comment:
create two different categories based on type please, for validator open p2p, prometheus, for full node open p2p, prometheus, rpc